fix(docs) Resolve 196 mdx lint warnings for just, quickly, actually, PostgreSQL (#47358)
supabase/supabase: apps/docs/content/guides/database/vault.mdx

### Updating secrets

> A secret can be updated with the `vault.update_secret()` function, this function makes updating secrets easy: provide the secret UUID as the first argument, and then an updated secret, updated optional unique name, or updated description:

Author: ```suggestion
To update a secret, use the `vault.update_secret()` function. Provide the secret UUID as the first argument, followed by an updated secret, name, or description:
```
